olp-cpp-sdk-dataservice-read/src/VersionedLayerClientImpl.cpp

```diff
@@ -790,6 +790,53 @@ bool VersionedLayerClientImpl::Release(const TileKeys& tiles) {
 
   return settings_.cache->Release(keys_to_release);
 }
+cache::KeyValueCache::KeyListType
+VersionedLayerClientImpl::GetRelatedPartitionKeys(
```

Comment:
Do you really need an extra method for this? Seems pretty basic and can be integrated into `Protect`

Reply:
it is just common code, which the same for protect and release